src/igvfd/schemas/changelogs/workflow.md

```diff
@@ -2,6 +2,7 @@
 
 ### Minor changes since schema version 5
 
+* Add `analysis_step_version`
```

Comment:
Missing period at the end and the property name should be plural analysis_step_versions.